src/commcare_cloud/ansible/roles/cloudwatch_logs/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Check if cloudwatch-agent-ctl exists
```

Comment:
This task fails if the agent does not exist. I just encountered that on a staging VM. Is there a way to install it?

Reply:
Do you know if this is because that instance was just created? We use the AWS Systems Manager to install the cloudwatch agent and I was under the impression that it ran on instance creation. If this is not the case, we should look to see if that is an option. 
I'll offline with you about how to run it manually.

Reply:
@knguyen142 I've created a ticket with the details here: https://dimagi-dev.atlassian.net/browse/SAAS-11469

I'm not sure when these VMs were created.